app/enterprise/0.34-x/kong-manager/configuration/networking.md

```diff
@@ -68,6 +68,10 @@ Emails from Kong Manager require the following configuration:
 * [`admin_emails_reply_to`](/enterprise/{{page.kong_version}}/property-reference/#admin_emails_reply_to)
 * [`admin_invitation_expiry`](/enterprise/{{page.kong_version}}/property-reference/#admin_invitation_expiry)
 
+⚠️**Important:** If the SMTP settings are configured incorrectly, 
```

Comment:
These don't look like SMTP settings - for SMTP I'd expect things like `host` and `port`

Also... I wonder if this warning is even necessary - would anyone expect Kong to check if email addresses are "valid" and functional? If you enter non-functional email addresses, the emails from Kong Manager still get sent, right? It is just that the recipient of the email cannot reply to them and/or cannot understand who they come from. I don't see that that needs any warning, it seems sufficiently obvious.